lib/plugins/create/templates/aws-fsharp/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#build handlers
+dotnet restore
+dotnet publish -c release
+
+#install zip
+apt-get -qq update
+apt-get -qq -y install zip
+
+#create deployment package
+pushd bin/Release/netcoreapp1.0/publish
```

Comment:
I think we should name the zip `deploy-package.zip` for consistancy with the other templates